src/scenes/UserGalleryPage/UserGalleryCollections.tsx

```diff
@@ -157,7 +157,10 @@ function UserGalleryCollections({ galleryRef, queryRef, mobileLayout }: Props) {
                   rowRenderer={rowRenderer}
                   style={{
                     outline: 'none',
+                    overflowX: 'visible',
+                    overflowY: 'visible',
                   }}
+                  containerStyle={{ overflow: 'visible' }}
```

Comment:
this is to make sure the icon doesn't get cut off anymore
<img width="1188" alt="image" src="https://user-images.githubusercontent.com/6754223/214369108-b5bf2e4c-2eae-42c4-98b3-7fcd013facbb.png">